src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -484,10 +484,9 @@ function _create_static_fitness_graph!(g::AbstractGraph, m::Integer, cum_fitness
         target = searchsortedfirst(cum_fitness_in, rand(rng) * max_in)
         # skip if loop edge
         (source == target) && continue
-        edge = SimpleEdge(source, target)
         # is there already an edge? If so, try again
-        add_edge!(g, edge) || continue
-        m -= 1
+        add_edge!(g, source, target) || continue
```

Comment:
`add_edge!()` should prefer `Edge` as an argument.

Reply:
But in the original code, we had the line `edge = SimpleEdge(source, target)` just so we could insert it, which seems to be unnecessary.